packages/components/nodes/chains/VectaraChain/VectaraChain.ts

```diff
@@ -247,17 +248,30 @@ class VectaraChain_Chains implements INode {
             lexicalInterpolationConfig: { lambda: vectaraFilter?.lambda ?? 0.025 }
         }))
 
+        const mmrRerankerId = 272725718 // Vectara reranker ID for MMR
```

Comment:
can we add a comment above to the docs link - https://docs.vectara.com/docs/api-reference/search-apis/reranking, as in why the hardcoded value